src/services/types.ts

```diff
@@ -130,3 +136,11 @@ export type ReferrerReward = {
   timestamp: number;
   amount: string;
 };
+
+export type ReferrerSetReferrer = {
+  referrer: string;
```

Comment:
I guess you should define in this type this structure of `historyElement` `data`
https://github.com/sora-xor/sora-subquery/blob/develop/src/mappings/setReferralHandler.ts#L16

```
from: string; // referral
to: string; // referrer
```